src/story-controls.ts

```diff
@@ -242,10 +255,10 @@ export type OptionsListType<T = unknown> = { [key: string]: T } | OptionsValueTy
  * 3. array of key-value pair objects
  */
 
-export interface StoryPropertyOptions<T = unknown> extends StoryPropertyBase<OptionsValueType<T>> {
-  type: PropertyTypes.OPTIONS;
+export interface StoryControlOptions<T = unknown> extends StoryControlBase<OptionsValueType<T>> {
+  type: ControlTypes.OPTIONS;
 
-  options: OptionsListType<T>;
+  options: OptionsListType;
```

Comment:
Why've removed generic for options?

Reply:
thanks, good catch. Just a mistake as I added manually the changes, I am really not great at git stuff.

Can you give it another look, please and confirm